frontend/src/repository.ts

```diff
@@ -19,13 +19,20 @@ export const defaultValues = {
   electricityWater: 18.4,
   electricityCoal: 117.72,
   electricityBiomass: 47.15,
-  electricityNuclear: 60.92
+  electricityNuclear: 60.92,
+
+  // 2020, https://www.umweltbundesamt.de/daten/klima/treibhausgas-emissionen-in-deutschland#nationale-und-europaische-klimaziele
+  co2emissionsIndustry: 186,
+  co2emissionsBuildings: 118,
+  co2emissionsMobility: 150,
+  co2emissionsAgriculture: 70,
+  co2emissionsOthers: 9
 }
 
 export const initialGame = {
   currentYear: 2021,
   values: defaultValues,
-  acceptedLaws: [],
+  acceptedLaws: []
```

Comment:
```suggestion
  acceptedLaws: [],
```
Ist es eher guter Stil, das Komma wegzulassen oder es schon zu setzen?
Ich kenne eher Sprachen, die einem die Wahl nicht lassen. Fänds ja ganz praktisch, wenn man keinen Diff bekommt nur wegen des Kommas, wenn man eine Zeile hinzufügen möchte.

Reply:
Ah! Sehe gerade, dass der "prettier" Formatter die Kommas setzt. Einfach den benutzen und nicht drüber nachdenken, wa?

Reply:
Ja, genau, das ist die Idee beim Prettier. Der sollte beim Speichern immer automatisch die Kommata dran machen (was hier wohl auch passiert ist), so dass bei der nächsten Änderung die Zeile nicht geändert werden muss.

Reply:
Obwohl, äh... das passiert offenbar gerade nicht... #mussNachschauen

Reply:
Ah, die Anpassung der `.prettierrc` kommt erst mit #9 rein.